src/core/initialize.ts

```diff
@@ -155,9 +155,6 @@ async function cloneProject({
       .move(
         path.join(directory, '_gitignore'),
         path.join(directory, '.gitignore'),
-        {
-          overwrite: true,
-        },
       )
```

Comment:
If the directory is not empty, the code cannot be executed here, and the logic here can be removed.